src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -170,7 +164,6 @@ public function __construct(
         TokenStorageInterface $tokenStorage = null,
         SecurityCheckerInterface $securityChecker = null,
         FFProbe $ffprobe = null,
-        $permissions,
```

Comment:
Removing this would be a bc break if somebody override the service and had called `parent::__construct` and expect permissions to given in its service definition.

Reply:
I would have lived with that... Because it feels very weird to me if a constructor parameter is set and never used. But fine, going to add it again.